jetstream/analysis.py

```diff
@@ -21,6 +21,33 @@
 from jetstream.dryrun import dry_run_query
 from jetstream.statistics import Count, StatisticResult, StatisticResultCollection
 
+logger = logging.getLogger()
```

Comment:
```suggestion
_logger = logging.getLogger(__name__)
```

(applicable everywhere; creating them with `__name__` is probably a useful idiom and I don't think we ever want to export them)